packages/plugin-http-client/src/rest/mapper/Mapper.ts

```diff
@@ -0,0 +1,9 @@
+export class Mapper {
+  deserialize(value: any) {
+    return value;
+  }
+
+  serialize(value: any) {
```

Comment:
`serialize` and `deserialize` can be optional? Also both methods are missing return type (which is probably `any` anyway :D)

Reply:
Added return type. Serialize and deserialize are mandatory for correct Entity serialize and deserialize.